examples/dfem/minimal_surface.cpp

```diff
@@ -0,0 +1,518 @@
+//                       MFEM Example - Minimal Surface
+//
+// Compile with: make minimal_surface
+//
+// Sample runs:  mpirun -np 4 minimal_surface -der 0
```

Comment:
This sample run fails as written because the ParaView DataCollection attempts to create a directory named "minimal_surface" but, of course, the executable already exists in the current directory with this name. Changing the name of the ParaView output directory should fix this.